api/jobset/v1alpha2/jobset_types.go

```diff
@@ -94,7 +94,18 @@ type JobSetSpec struct {
 	// Suspend suspends all running child Jobs when set to true.
 	Suspend *bool `json:"suspend,omitempty"`
 
-	// ManagedBy is used to indicate the controller or entity that manages a JobSet
+	// ManagedBy is used to indicate the controller or entity that manages a JobSet.
+	// The built-in JobSet controller reconciles JobSets which don't have this
+	// field at all or the field value is the reserved string
+	// `jobset.sigs.k8s.io/jobset-controller`, but skips reconciling JobSets
+	// with a custom value for this field.
+	//
+	// The value must be a valid domain-prefixed path (e.g. acme.io/foo) -
+	// all characters before the first "/" must be a valid subdomain as defined
+	// by RFC 1123. All characters trailing the first "/" must be valid HTTP Path
+	// characters as defined by RFC 3986. The value cannot exceed 63 characters.
+	// The field is immutable.
+	// +optional
 	ManagedBy *string `json:"managedBy,omitempty"`
```

Comment:
This is just curious. Why we don't use CEL validation instead of webhook validation here?
https://github.com/kubernetes-sigs/jobset/blob/7752a365fa5c9ab497c5b61d4dcd7c9e3609e9cb/pkg/webhooks/jobset_webhook.go#L231

Reply:
No specific reason for using webhook validation here. We can look into migrating to CEL as a follow up.

Reply:
I see. Thank you.